applications/FluidDynamicsApplication/tests/cpp_tests/test_estimate_dt_utilities.cpp

```diff
@@ -0,0 +1,108 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "testing/testing.h"
+#include "containers/model.h"
+#include "includes/cfd_variables.h"
+
+// Application includes
+#include "fluid_dynamics_application_variables.h"
+#include "custom_utilities/estimate_dt_utilities.h"
+
+namespace Kratos {
+namespace Testing  {
+
+namespace Internals {
+
+void TestEstimateDtUtilitiesInitializeModelPart(
+    ModelPart& rModelPart,
+    const double DeltaTime)
+{
+
+    rModelPart.AddNodalSolutionStepVariable(VELOCITY);
+    Properties::Pointer p_properties = rModelPart.CreateNewProperties(0);
+
+    // Geometry creation
+    rModelPart.CreateNewNode(1, 0.0, 0.0, 0.0);
+    rModelPart.CreateNewNode(2, 2.0, 0.0, 0.0);
+    rModelPart.CreateNewNode(3, 0.0, 1.0, 0.0);
+    rModelPart.CreateNewNode(4, 1.0, 1.0, 0.0);
+    std::vector<ModelPart::IndexType> element_nodes_1{1, 2, 3};
+    std::vector<ModelPart::IndexType> element_nodes_2{3, 2, 4};
+    rModelPart.CreateNewElement("QSVMS2D3N", 1, element_nodes_1, p_properties);
+    rModelPart.CreateNewElement("QSVMS2D3N", 2, element_nodes_2, p_properties);
```

Comment:
cannot you use baseelements here?
I thought the CFL-cals is independent?

=> if you ever change or even remove the `QSVMS2D3N` then you would also need to change this test which is completely unrelated

Reply:
I think this should be possible. Thanks for the hint!

Reply:
Done!